docs/usage/directives/configurations/shutdown-time.md

```diff
@@ -0,0 +1,12 @@
+# Shutdown time
+
+`ShutdownTime=[ ISO8601_TIME | none ]`
+
+Allows to set a specific time in the global section when the instances should shutdown.
```

Comment:
Let's specify on here that system will only shutdown if the systemclock is properly in sync with an NTP server.

Reply:
Okay, added.